stylesheet.css

```diff
@@ -16,7 +16,7 @@ textarea:focus {
 
 header {
   background: #e5f3f2;
-  height: auto;
+  height: 37vh;
   width: 100%;
```

Comment:
VH and % and em and all those relative units of size can have a bunch of unintended consequences when you're looking at stuff on different screens and browsers. Are you sure you can't do this with `px`?